SEP-0009.md

```diff
@@ -0,0 +1,106 @@
+# SEP-0009 -- Release Pattern for SunPy Core
+
+| SEP           | 9                              |
+|---------------|--------------------------------|
+| title         | Release Pattern for SunPy Core |
+| author(s)     | Stuart Mumford, Russell Hewett  |
+| contact email | stuart@cadair.com              |
+| date-creation | 2018-12-13                     |
+| type          | informational                  |
+| status        | discussion                     |
+| discussion    |                                |
+
+# Introduction
+
+This SEP introduces a predictable release pattern for the SunPy core package so
+that packages which have SunPy as an upstream dependency can plan their upgrades
+and changes in a transparent way. This SEP introduces timed releases and a
+versioning system.
+
+SunPy is being used more as an upstream dependency for both affiliated packages
+and packages in the wider solar or heliophysics ecosystems. A predictable
+release and support cycle for the SunPy core package will aid these packages in
+planning their own development cycle.
+
+The desire for stability and predictability is countered by the desire of the
+SunPy developers to continue providing new features and functionality and
+continue to iterate on the SunPy API to further increase our usability and
+correctness.
+
+This SEP therefore tries to focus on when the SunPy core package will be
+released rather than placing any strict limitations on changes and API
+stability.
+
+# Detailed Description
+
+## Release Schedule
+
+This SEP proposes that the SunPy core package will be released at least twice a
+year, initially a release in the months of May and November have been chosen to
+place the SunPy release shortly after that of the Astropy core package. The
+decision to make a release and the specific timing of releases are at the
+discretion of the lead developer, and changes to the months of the releases can
+be made without modifying this SEP to follow any pertinent release cycles of
+upstream packages or any other reason.
+
+Future changes to the LTS release cycle should be informed by upstream package
+development. It is important that upstream packages versions are supported for
+the lifetime of a SunPy LTS.
+
+## Supporting Releases with bug fixes
+
+This SEP proposes that there be two categories of SunPy release, a LTS (Long
+Term Support) release and a normal or non-LTS release. The LTS release will have
+an extended length of bug fixes provided by the SunPy developers. We propose
+that the first release in a year be a non-LTS release, which will be supported
+for 6 months (until the next release), the second release in a year will be an
+LTS release and will be supported for 12 months (until the next LTS release).
+
+The objective of this support cycle is to balance the desire to provide a
+release cycle for people who wish to experience minimal changes to SunPy with
+minimizing the maintenance burden on the SunPy developers. It is feasible that
```

Comment:
Suggest to remove sentence starting from "It is feasible..." to the end of paragraph. May not age well.